helpers/achievements.ts

```diff
@@ -0,0 +1,98 @@
+export const achievements = [
```

Comment:
Can we use `import { useTranslations } from "next-intl";` here?
Also, id 4 and 5 is duplicated.

IMO api should return a list of ids that the user have achieved.
That way we keep the images here in the app, like you did.